citus-enterprise.spec

```diff
@@ -302,6 +305,7 @@ done < "$secret_files_list"
 %files
 %defattr(-,root,root,-)
 %doc CHANGELOG.md
+%doc vendor/safestringlib/LICENSE
 %if 0%{?rhel} && 0%{?rhel} <= 6
 %doc LICENSE
```

Comment:
Apparently we need to provide the license file in a `%license` field (except for el/6). This license still contains only Citus license. Should we introduce a new `PACKAGE-LICENSING` file here? See excerpt from [FedoraProject Licensing Guide](https://docs.fedoraproject.org/en-US/packaging-guidelines/LicensingGuidelines/#_multiple_licensing_scenarios) below.

>Including a file as %license which contains the licensing breakdown for the packaged files, then using:
>
>\# For a breakdown of the licensing, see PACKAGE-LICENSING

Reply:
hmm, then should we concat two licenses in a new file `PACKAGE-LICENSING` and then put it with `%license` ?

Reply:
(when it is a debian package)

Reply:
or I guess below is also good:

```
%if 0%{?rhel} && 0%{?rhel} <= 6
%doc LICENSE
%doc vendor/safestringlib/LICENSE
%else
%license LICENSE
%license vendor/safestringlib/LICENSE
%endif
```

Reply:
```
+ cp -pr LICENSE /root/rpmbuild/BUILDROOT/citus-enterprise93_11-9.3.3.citus-1.el6.x86_64/usr/share/doc/citus-enterprise93_11-9.3.3.citus
+ cp -pr vendor/safestringlib/safestringlib-LICENSE /root/rpmbuild/BUILDROOT/citus-enterprise93_11-9.3.3.citus-1.el6.x86_64/usr/share/doc/citus-enterprise93_11-9.3.3.citus
+ exit 0
    File must begin with "/": LANG=C
    File must begin with "/": export
    File must begin with "/": LANG
    File must begin with "/": unset
    File must begin with "/": DISPLAY
```

I am not sure about the root cause of this issue. I can think of 2 things:
1- Having two `%license` fields are not allowed
2- The path for safestringlib license is broken.

With the latest commits however, I think the license path is now fixed.

Reply:
> hmm, then should we concat two licenses in a new file PACKAGE-LICENSING and then put it with %license

I think this makes sense.